CHANGELOG.md

```diff
@@ -6,6 +6,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [30.0.0]
+### Uncategorized
+- Introduce Detected Tokens ([#808](https://github.com/MetaMask/controllers/pull/808))
+- Always pull from dynamic token list from TokenListController ([#806](https://github.com/MetaMask/controllers/pull/806))
+
+### Added
+- **BREAKING:** Expose chainId to AssetsContractController and add check whether current network supports token detection ([#809](https://github.com/MetaMask/controllers/pull/809))
+  - Consumers will need to pass listener method `onNetworkStateChange` in `AssetsContractController` constructor options object. This method should be called with network/provider details when network changes occur.
+- Add "onCollectibleAdded" event handler to the CollectiblesController constructor ([#814](https://github.com/MetaMask/controllers/pull/814))
+  - This event handler was added to allow us to capture metrics.
+- Append phishfort blocklist to list used by PhishingController ([#715](https://github.com/MetaMask/controllers/pull/715))
```

Comment:
This should be listed under "changed", and it includes breaking changes. Specifically the `test` method now returns a different result. We'll also want to include advise about updating designs to ensure we don't direct people disputing a Phishfort block to make an issue on our `eth-phishing-detect` repo (as our current designs would do if they were unaltered after this update).

Reply:
added the note about breaking change here:https://github.com/MetaMask/controllers/pull/848/commits/5114337b3812caf7511456fabc63215bd7e45e73.

> We'll also want to include advice about updating designs to ensure we don't direct people disputing a Phishfort block to make an issue on our eth-phishing-detect repo

not clear on what this would look like.

Reply:
Maybe something like:

```
  - Designs may need to be updated to account for the fact that sites may now be blocked by Phishfort. We should ensure users are not directed to the `eth-phishing-detect` repository to dispute Phishfort blocks, because we will not be able to help them.
```

Reply:
I created a ticket for handing this on `phishing-warning` as well: https://github.com/MetaMask/phishing-warning/issues/15
cc @mcmire @danjm

Reply:
add your comment here: https://github.com/MetaMask/controllers/pull/848/commits/87da72b1fae8ec3b18c866812897e725e95b44b8